internal/runners/initialize/init.go

```diff
@@ -97,8 +98,32 @@ func inferLanguage(config projectfile.ConfigGetter) (string, string, bool) {
 }
 
 func (r *Initialize) Run(params *RunParams) (rerr error) {
-	defer rationalizeError(params.Namespace, &rerr)
-	logging.Debug("Init: %s/%s %v", params.Namespace.Owner, params.Namespace.Project, params.Private)
+	logging.Debug("Init: %s %v", params.Namespace, params.Private)
+
+	var (
+		paramOwner          string
+		paramProjectName    string
+		resolvedOwner       string
+		resolvedProjectName string
+	)
+
+	if params.Namespace != "" {
+		ns, err := project.ParseNamespace(params.Namespace)
+		if err != nil {
+			// If the namespace was invalid but an argument was passed, we
+			// assume it's a project name and not an owner.
+			logging.Error("Could not parse namespace: %v", err)
+			paramProjectName = params.Namespace
+		} else {
+			paramOwner = ns.Owner
+			paramProjectName = ns.Project
+
+		}
+	}
```

Comment:
I feel like cmdtree/init.go's `Run()` function should be interpreting parsed parameters and correctly filling in `RunParams` so that the runner doesn't have to do this kind of interpretation.